src/news/news.js

```diff
@@ -0,0 +1,51 @@
+import './news.scss';
+
+export default class News {
+  constructor () {
+    this.showNews = this.showNews.bind(this);
+  }
+
+  async getNewsAsync(url) {
```

Comment:
getNewsAsync(url)
showNews (event)

the space between function name and arguments list is not consistent in your code.
Suggest to add inter to the project
It is not required, up to your taste